pkg/plugins/golang/declarative/v1/api.go

```diff
@@ -19,25 +19,20 @@ package v1
 import (
 	"errors"
 	"fmt"
-	"path/filepath"
-
-	"github.com/spf13/afero"
 
 	"sigs.k8s.io/kubebuilder/v3/pkg/config"
 	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
 	"sigs.k8s.io/kubebuilder/v3/pkg/model/resource"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin/util"
-	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/declarative/v1/internal/templates"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang/declarative/v1/scaffolds"
```

Comment:
The scaffolds need to be inside of `scaffolds/internal/templates` for we follow up the same structure of https://github.com/kubernetes-sigs/kubebuilder/tree/master/pkg/plugins/golang/v3/scaffolds

Also, see that we do not want to export the templates. So, could you please do this small change @viveksyngh ?

Reply:
Sorted out.